Deploy/Build-HelpFiles.ps1

```diff
@@ -35,7 +35,7 @@ function Remove-CommonParameterFromMarkdown {
     $ErrorActionPreference = 'Stop'
     $Docs = Get-ChildItem -Path $Path -Recurse
     foreach ($p in $Docs) {
-        Write-Host "[status]Removing ProgressAction from $p"
+        Write-Host "[status] Removing ProgressAction from $p"
```

Comment:
Only reason for this change is tech debt, I wanted to see all these status messages appear similar. Some had spaces, others didn't.